desktop/caliban.py

```diff
@@ -3092,8 +3076,19 @@ def __init__(self, filename, raw, annotated, save_vars_mode):
         # file opens to the first channel
         self.channel = 0
 
+        # should be robust to 3D, 4D, and some cases of 5D array sizes
+        self.dims = raw.ndim
+        if self.dims == 3:
+            self.raw = np.expand_dims(self.raw, axis=0)
+            self.annotated = np.expand_dims(self.annotated, axis=0)
+
+        elif self.dims == 5:
+            self.raw = np.squeeze(self.raw, axis=0)
+            self.annotated = np.squeeze(self.annotated, axis=0)
+
```

Comment:
Yeah this makes sense, we should probably let the user know if we're making assumptions about their data.

Reply:
"Raw and annotated images must both be in 4D arrays in the shape (frames, y, x, channels or features)." is right at the top of the README--so I do feel like we are letting them know we've made that assumption of the data. 3D/5D opening is more to catch situations where a user got this wrong due to a trivial dimension (I've done this before when breaking up larger datasets into npzs for Caliban). I don't think we need to add additional messaging to the program itself, although I wouldn't be opposed to slightly modifying that part of the README.

Reply:
Not all users read docs or REAMEs, and it would still be a useful alert. In your case, it would have let you know you've broken up the dataset incorrectly, which could cause other errors downstream. No need to change the README, but we should alert the user when we transform data. A prompt may be too much, but a logging statement would be appropriate.